fbw-common/src/systems/shared/src/publishers/Extras/ExtrasSimVarPublisher.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023 FlyByWire Simulations
```

Comment:
I don't think we should add any more of these "publish all sorts of random stuff" publishers. Each publisher should just publish the topics for one subsystem.

Reply:
I understand that and was wondering about that, in this case though unless I misunderstand I would end up with like 3 single var publishers. Or do you mean putting all the vars that I use for the GPU stuff into one publisher ?

Reply:
They grow over time as code gets refactored to improve quality and new code written. There's no problem with having one var at the start. It doesn't make sense to duplicate simvar definitions over and over in every instrument.

For the MSFS ones there's https://github.com/flybywiresim/aircraft/blob/master/fbw-common/src/systems/shared/src/publishers/Msfs/. Pay careful attention to the JSDocs in the existing ones for how they should be setup... everything should exactly match the SDK docs, including naming, units, and which publisher things go in. That way it's easy to find things, and there's no room for debate, and with the msfs_ prefix no chance of collisions with our events.
```ts
/**
 * Events for simvars listed on https://docs.flightsimulator.com/html/Programming_Tools/SimVars/Aircraft_SimVars/Aircraft_AutopilotAssistant_Variables.htm.
 * Event names are the same as the simvar names, with msfs_ prefix, and index as suffix for indexed simvars.
 */
 ```